pkg/sqlite/image.go

```diff
@@ -239,3 +240,45 @@ func translateAnnotationsIntoPackage(annotations *registry.AnnotationsFile, csv
 
 	return manifest, nil
 }
+
+// ImageLoader loads a bundle image of resources into the database
+type ImageExporter struct {
+	store         registry.Query
+	image         string
+	directory     string
+	containerTool string
+}
+
+func NewSQLExporterForImage(store registry.Query, image, directory, containerTool string) *ImageExporter {
+	return &ImageExporter{
+		store:         store,
+		image:         image,
+		directory:     directory,
+		containerTool: containerTool,
+	}
+}
+
+func (i *ImageExporter) Export() error {
+
+	log := logrus.WithField("img", i.image)
+
+	version, err := i.store.GetBundleVersion(context.TODO(), i.image)
```

Comment:
What if there is no version? Fallback to the bundle name?

Wouldn't it be easier to just use i.image (appropriately munged to be a filename) as the folder? The loader won't care what the folders are named.